modules/tracing/mule-tracer-customization-impl/src/main/java/org/mule/runtime/tracer/customization/impl/provider/DebugInitialExportInfoProvider.java

```diff
@@ -33,10 +33,12 @@
 /**
  * An {@link InitialExportInfoProvider} corresponding to the {@link TracingLevel#DEBUG}
  *
- * @since 4.6.0
+ * @since 4.5.0
  */
 public class DebugInitialExportInfoProvider extends MonitoringInitialExportInfoProvider {
 
+  private boolean isOverride = false;
```

Comment:
Is this correct? The debug initialExportInfoProvider provides exportInfo which are the one retrieved for each location. It is used for every exportSpanInfo.